src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -10,24 +10,29 @@
 using System.Reflection;
 using Microsoft.Cci;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.Symbols.SynthesizedAutoPropAccessorSymbol;
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
-    internal sealed class SynthesizedRecordPropertySymbol : SourceOrRecordPropertySymbol
+    internal sealed class SynthesizedRecordPropertySymbol : SourcePropertySymbolBase
```

Comment:
>SourcePropertySymbolBase [](start = 60, length = 24)

What non-trivial support are we trying to get from the base class here? If none, then I think we should inherit directly from PropertySymbol.